src/fireflyClient.ts

```diff
@@ -1220,6 +1241,349 @@ export class FireflyClient {
     return userAuthToken;
   };
 
+  /**
+   * Reset timer for cancel on disconnect for open orders
+   * @param params PostTimerAttributes containing the countdowns of all markets
+   * @returns PostTimerResponse containing accepted and failed countdowns. If status is not 201, request wasn't successful.
+   */
+  resetCancelOnDisconnectTimer = async (params: PostTimerAttributes) => {
+    const response = await this.apiService.post<PostTimerResponse>(
+      SERVICE_URLS.USER.CANCEL_ON_DISCONNECT,
+      params,
+      { isAuthenticationRequired: true },
+      this.network.dmsURL
+    );
+    if (response.status == 503) {
+      throw Error(`Cancel on Disconnect (dead-mans-switch) feature is currently unavailable`);
+
+    }
+    return response;
+  };
+
+  /**
+ * Gets user Cancel on Disconnect timer
+ * @returns GetCountDownsResponse
+ */
+  getCancelOnDisconnectTimer = async (symbol?: string, parentAddress?: string) => {
+    const response = await this.apiService.get<GetCountDownsResponse>(
+      SERVICE_URLS.USER.CANCEL_ON_DISCONNECT,
+      {
+        parentAddress,
+        symbol
+      },
+      { isAuthenticationRequired: true },
+      this.network.dmsURL
+    );
+    if (response.status == 503) {
+      throw Error(`Cancel on Disconnect (dead-mans-switch) feature is currently unavailable`);
+    }
+    return response;
+  };
+
+    /**
+   * Places a signed order on firefly exchange
+   * @param params PlaceOrderRequest containing the signed order created using createSignedOrder
+   * @returns PlaceOrderResponse containing status and data. If status is not 201, order placement failed.
+   */
+    placeSignedOrderV2 = async (params: PlaceOrderRequest) => {
+      const response = await this.apiService.post<PlaceOrderResponse>(
+        SERVICE_URLS.V2_ORDERS.ORDERS,
+        {
+          symbol: params.symbol,
+          userAddress: params.maker,
+          orderType: params.orderType,
+          price: toBigNumberStr(params.price),
+          triggerPrice: toBigNumberStr(params.triggerPrice || "0"),
+          quantity: toBigNumberStr(params.quantity),
+          leverage: toBigNumberStr(params.leverage),
+          side: params.side,
+          reduceOnly: params.reduceOnly,
+          salt: params.salt,
+          expiration: params.expiration,
+          orderSignature: params.orderSignature,
+          timeInForce: params.timeInForce || TIME_IN_FORCE.GOOD_TILL_TIME,
+          postOnly: params.postOnly || false,
+          clientId: params.clientId
+            ? `firefly-client: ${params.clientId}`
+            : "firefly-client",
+        },
+        { isAuthenticationRequired: true }
+      );
+  
+      return response;
+    };
+  
+    /**
+     * Given an order payload, signs it on chain and submits to exchange for placement
+     * @param params PostOrderRequest
```

Comment:
@malikkulsoom @shahzaib-sheikh - I recommend not exposing these PostOrderV2 and corresponding calls like  placeSignedOrderV2  to the clients and partners. This endpoint could be used only internally for testing cache behavior, and it might confuse partners if we expose it. cc @MaazUddin11

Reply:
@singhalhemant they were already added, I just changed the placement of these methods. If you want I can remove these in another PR?